flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java

```diff
@@ -323,9 +323,10 @@ public float getProgress() {
 	}
 
 	@Override
-	public void onContainersCompleted(final List<ContainerStatus> list) {
+	public void onContainersCompleted(final List<ContainerStatus> statuses) {
 		runAsync(() -> {
-				for (final ContainerStatus containerStatus : list) {
+				log.debug("YARN ResourceManager reported the following containers completed: {}.", statuses);
```

Comment:
here will call the list's toString method? Is the format suitable?

Reply:
That depends on the `List` implementation. At least `java.util.AbstractCollection` has a meaningful string representation. The List returned by Hadoop 2.8 as well.